app/src/main/java/cs465/illinois/edu/dogonthequad/CreateMeetupReviewActivity.java

```diff
@@ -37,10 +46,44 @@ protected void onCreate(Bundle savedInstanceState) {
             b.setOnClickListener(new View.OnClickListener() {
                 @Override
                 public void onClick(View v) {
-                    gotoActivity(activity);
+                    gotoActivity(activity, true);
                 }
             });
         }
+
+        mEndTimeText = findViewById(R.id.end_time_text);
+        mSocialLevelText = findViewById(R.id.social_level_text);
+        mDogsText = findViewById(R.id.dogs_text);
+    }
+
+    @Override
+    protected void onStart() {
+        super.onStart();
+        if (mMeetup != null) {
+            if (mMeetup.mEndTime != null) {
+                String text = "End Time: " + mMeetup.mEndTime.toString();
+                mEndTimeText.setText(text);
+            }
+            if (mMeetup.mSocialLevel != null) {
+                switch (mMeetup.mSocialLevel) {
+                    case Low:
+                        mSocialLevelText.setText(R.string.social_level_low);
+                        break;
+                    case Medium:
+                        mSocialLevelText.setText(R.string.social_level_medium);
+                        break;
+                    case High:
+                        mSocialLevelText.setText(R.string.social_level_high);
+                        break;
+                }
+            }
+            if (mMeetup.mDogs != null) {
+                // TODO: get real names of dogs
+                List<String> names = mMeetup.mDogs.stream().map(UUID::toString).collect(Collectors.toList());
+                String text = "Dogs:\n" + TextUtils.join("\n ", names);
+                mDogsText.setText(text);
```

Comment:
Having more than one dog selected causes the text to display under the progress bar controls

Reply:
So for right now, I've "fixed" the problem using ellipses at the end of the line. Perhaps later we'll find a fix for this, but for now I'm ok with ellipses.